packages/core/services/maps-api-loader/lazy-maps-api-loader.ts

```diff
@@ -14,7 +14,7 @@ export enum GoogleMapsScriptProtocol {
  * Token for the config of the LazyMapsAPILoader. Please provide an object of type {@link
  * LazyMapsAPILoaderConfig}.
  */
-export const LAZY_MAPS_API_CONFIG = new InjectionToken('angular-google-maps LAZY_MAPS_API_CONFIG');
+export declare const LAZY_MAPS_API_CONFIG: InjectionToken<LazyMapsAPILoaderConfigLiteral>;
```

Comment:
I don't understand why that is needed. Can you explain?

Reply:
That line gives the error `[ts] A 'const' initializer in an ambient context must be a string or numeric literal.` when I perform the swap of `OpaqueToken` with `InjectionToken` in the file installed in `node_modules` in my project.

![typescript-ambient-context-error](https://user-images.githubusercontent.com/1382768/30522948-43257dbc-9ba6-11e7-866c-e729372d87b6.jpg)

Though when I open up this repository directly there is no error on that line.